### PR TITLE
add reverseproxy example using puppetlabs/apache module

### PR DESCRIPTION
**Pull request by kpankonen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,6 +213,22 @@ ReverseProxy /pboard/ http://dashes.acme:5000/pboard/
 ProxyPassReverse /pboard/ http://dashes.acme:5000/pboard/
 ```
 
+Using the puppetlabs/apache module:
+```puppet
```
Comment (bastelfreak) [inline README.md]: can you please add a newline between the next and the clodeblock? Otherwise a markdown linter complains

**Comment by bastelfreak** [comment]
Hi @kpankonen, thanks for the PR! Can you take a look at the inline comment I made?

**Comment by kpankonen** [comment]
newline added!

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**